ui/widgets/datepicker.js

```diff
@@ -61,12 +62,13 @@ var widget = $.widget( "ui.datepicker", {
 
 	_create: function() {
 		this.suppressExpandOnFocus = false;
+		this._parse = new Globalize( this.options.locale ).dateParser( this.options.dateFormat );
 
 		if ( $.type( this.options.max ) === "string" ) {
-			this.options.max = Globalize.parseDate( this.options.max, { raw: "yyyy-MM-dd" } );
+			this.options.max = this._parse( this.options.max );
```

Comment:
According to http://wiki.jqueryui.com/w/page/12137778/Datepicker both `min` and `max` should be `date`, not `string`. Is that document out of date?

If so, wasn't it on purpose that we used a fixed iso format for min/max as string, instead of matching the `dateFormat` option? With this change, the developer would need to update min/max whenever the (user facing) `dateFormat` option changes.